### PR TITLE
feat(config): add MyOT OpenTherm Actuator v2

### PR DESCRIPTION
**Pull request by czesiu20**
Configuration file for new device MyOT v2 (OpenTherm Actuator)

### CODE REVIEW

**Comment by czesiu20** [comment]
@zwave-js-bot fix lint

File: packages/config/config/devices/0x044c/myot2.json [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+			"$import": "~/templates/master_template.json#base_0-100_nounit",
+			"label": "Minimum Boiler Setpoint",
+			"unit": "°C",
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "3",
+			"$import": "~/templates/master_template.json#base_0-100_nounit",
+			"label": "Maximum DHW Setpoint",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x044c/myot2.json]: Is DHW something commonly known or should we use the full word here?
At least I don't know what is is 😅

File: packages/config/config/devices/0x044c/myot2.json [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+		{
+			"#": "20",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Pump State",
+			"readOnly": true
+		},
+		{
+			"#": "21",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Comp State",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x044c/myot2.json]:
```suggestion
			"label": "Compressor State",
```
or is that something else?

File: packages/config/config/devices/0x044c/myot2.json [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+		{
+			"#": "30",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "DHW Enable"
+		},
+		{
+			"#": "50",
+			"label": "Load Power",
+			"description": "The power of the load W x100 (inverter)",
+			"unit": "W",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x044c/myot2.json]: Should this be a unit of `0.01 W`, so a setting of 100 corresponds to 1W?
If so, please update the unit and remove the description.

File: packages/config/config/devices/0x044c/myot2.json [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+			"readOnly": true
+		},
+		{
+			"#": "51",
+			"label": "Battery Voltage Output",
+			"description": "The voltage output of the battery V x100 (inverter)",
+			"unit": "V",
+			"valueSize": 2,
+			"minValue": -32768,
+			"maxValue": 32767,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x044c/myot2.json]: Do negative values mean the battery is charging? Or is this one of these cases where the number is actually 0...65535 and needs to be re-calculated to match the signed representation Z-Wave has by default?

File: packages/config/config/devices/0x044c/myot2.json (hunk 3 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x044c/myot2.json]: Same for the following parameters

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x044c/myot2.json (hunk 1 above)
Comment (czesiu20) [reply packages/config/config/devices/0x044c/myot2.json]: We can change to the full name: Domestic Hot Water

File: packages/config/config/devices/0x044c/myot2.json (hunk 3 above)
Comment (czesiu20) [reply packages/config/config/devices/0x044c/myot2.json]: You're right. I haven't thought about it. This is multi purpose device for boilers, air conditioning or PV inverters. I'm using it for my boiler.

File: packages/config/config/devices/0x044c/myot2.json (hunk 4 above)
Comment (czesiu20) [reply packages/config/config/devices/0x044c/myot2.json]: Honestly, I have no idea:) I can see negative values in the configuration so i decided to use signed range. The documentation of the device is very poor. The device is still under development.

Comment (AlCalzone) [reply packages/config/config/devices/0x044c/myot2.json]: Ok we can change this when necessary and there's more information. Signed values are the default in Z-Wave, but many just re-interpret them as unsigned.

File: packages/config/config/devices/0x044c/myot2.json (hunk 1 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x044c/myot2.json]: I'd prefer that

File: packages/config/config/devices/0x044c/myot2.json (hunk 4 above)
Comment (czesiu20) [reply packages/config/config/devices/0x044c/myot2.json]: I'm trying to get this information from the manufacturer. I'll fix it when receive any information.

File: packages/config/config/devices/0x044c/myot2.json (hunk 1 above)
Comment (czesiu20) [reply packages/config/config/devices/0x044c/myot2.json]: Changed to the full name.

File: packages/config/config/devices/0x044c/myot2.json (hunk 3 above)
Comment (czesiu20) [reply packages/config/config/devices/0x044c/myot2.json]: Updated as requested.

File: packages/config/config/devices/0x044c/myot2.json (hunk 4 above)
Comment (czesiu20) [reply packages/config/config/devices/0x044c/myot2.json]: Due to your findings that the unit is 0.01, an unsigned value is more likely. I'm still waiting for any information from the manufacturer. Updated to the suggested range.

**Comment by czesiu20** [comment]
@zwave-js-bot fix lint

File: packages/config/config/devices/0x044c/myot2.json [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+			"valueSize": 2,
+			"readOnly": true
+		},
+		{
+			"#": "53",
+			"$import": "~/templates/master_template.json#base_0-65535",
+			"label": "Battery Power Output",
+			"unit": "0.01 W",
+			"readOnly": true
+		},
```
Comment (AlCalzone) [inline packages/config/config/devices/0x044c/myot2.json:143]: Is it on purpose that some of these params set valueSize to 2, and some leave it on 4 (from the $import)?

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Looks good, just a question

File: packages/config/config/devices/0x044c/myot2.json (hunk 5 above)
Comment (czesiu20) [reply packages/config/config/devices/0x044c/myot2.json:143]: Yes, accordingly to the unofficial document received from the manufacturer.
[MyOT v3.pdf](https://github.com/zwave-js/node-zwave-js/files/9021342/MyOT.v3.pdf)
.

**Review by AlCalzone — APPROVED** [review]
OK with my changes

**Pull request merged**